app/models/user.rb

```diff
@@ -1,9 +1,9 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments
-  has_many :likes
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
 
   # Validation: Name must not be blank
   validates :name, presence: true
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly, you should create your `user.admin?` method inside the user model. like below 🙏🏼 

```ruby
...
   def admin?
     role == 'admin' ? true : false
   end
```

Reply:
@mukhammadrizooff Thank you for letting me know, I've already made the changes